changelogs/fragments/26937-fix-remove-orphans.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - docker_compose - fixed an issue where remove_orphans doesn't work reliably
```

```suggestion
  - docker_compose - fixed an issue where ``remove_orphans`` doesn't work reliably.
```
Same for the other PR.

---

You should update the changelog entry. Especially the ReST formatting is really needed.